src/System.Management.Automation/DscSupport/ConvertCimMofToJsonCommand.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.PowerShell.DesiredStateConfiguration.Internal
+{
+    /// <summary>
+    /// Convert-CimMofToJson cmdlet implementation
+    /// </summary>
+    [Cmdlet(VerbsData.Convert, "CimMofToJson")]
```

Comment:
Add a HelpInfoUrl for the cmdlet

Reply:
File was removed - cmdlet was moved out of SMA dll to the PSDesiredStateConfiguration module.